test/integration_test.sh

```diff
@@ -286,32 +293,40 @@ function stop_lira_on_error {
 trap "stop_lira_on_error" ERR
 
 # 8. Send in notifications
+printf "\n\nGetting notification token\n"
+
+notification_token=$(docker run -i --rm \
+      -e VAULT_TOKEN=$vault_token \
+      broadinstitute/dsde-toolbox \
+      vault read -field=notification_token secret/dsde/mint/$env/listener/listener_secret)
+
 printf "\n\nSending in notifications\n"
-secrets_json_suffix=$(basename $secrets_json)
-#tenx_workflow_id=$(docker run --rm -v $script_dir:/app \
-#                    -e LIRA_URL="http://lira:8080/notifications" \
-#                    -e SECRETS_FILE=/app/$secrets_json_suffix \
-#                    -e NOTIFICATION=/app/10x_notification_${env}.json \
-#                    --link lira:lira \
-#                    broadinstitute/python-requests /app/send_notification.py)
 ss2_workflow_id=$(docker run --rm -v $script_dir:/app \
                     -e LIRA_URL="http://lira:8080/notifications" \
-                    -e SECRETS_FILE=/app/$secrets_json_suffix \
-                    -e NOTIFICATION=/app/ss2_notification_${env}.json \
+                    -e NOTIFICATION_TOKEN=$notification_token \
+                    -e NOTIFICATION=/app/ss2_notification_dss_staging.json \
                     --link lira:lira \
                     broadinstitute/python-requests /app/send_notification.py)
 
-#printf "\ntenx_workflow_id: $tenx_workflow_id"
 printf "\nss2_workflow_id: $ss2_workflow_id"
 
 # 9. Poll for completion
 printf "\n\nAwaiting workflow completion\n"
 
+export cromwell_user=$(docker run -i --rm \
+      -e VAULT_TOKEN=$vault_token \
+      broadinstitute/dsde-toolbox \
+      vault read -field=cromwell_user secret/dsde/mint/$env/common/htpasswd)
+
+export cromwell_password=$(docker run -i --rm \
+      -e VAULT_TOKEN=$vault_token \
+      broadinstitute/dsde-toolbox \
+      vault read -field=cromwell_password secret/dsde/mint/$env/common/htpasswd)
+
 python $script_dir/await_workflow_completion.py \
   --workflow_ids $ss2_workflow_id \
   --workflow_names ss2 \
   --cromwell_url https://cromwell.mint-$env.broadinstitute.org \
```

Comment:
The `cromwell_user` and `cromwell_password` should get passed into `await_workflow_completion.py` here

Reply:
These are getting set and read from environment variables.

Reply:
Oh I just noticed the `await_workflow_completion.py` script gets the `cromwell_user` and `cromwell_password` from the environment variables by default